src/main/java/com/mael/ttt/Game.java

```diff
@@ -16,42 +15,24 @@
     private String playerMark;
     private String opponentMark;
 
-    public Game(Board board, UserInterface gameUI) {
+    public Game(Board board, UserInterface gameUI, List<Player> players) {
         this.board   = board;
         this.gameUI  = gameUI;
+        player1      = players.get(0);
+        player2      = players.get(1);
+
         playerMark   = Marks.PLAYER.toString();
```

Comment:
I would use the `Marks` enum for as long as you can and only convert them to Strings when you absolutely have to. 
Enums can be compared with ``==`` and are more expressive than a String. A String is great for textual values (names, messages, notes,...) whereas enum are better for "_things with a defined, small set of possible variations_".
Your `Marks` can be **exactly** one of two things: PLAYER or OPPONENT. The String version would already be different if you just changed the casing of a single letter.